src/flagon/backends/datastore.py

```diff
@@ -0,0 +1,89 @@
+# The MIT License (MIT)
+#
+# Copyright (c) 2016 Fabio Franco Uechi
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to
+# deal in the Software without restriction, including without limitation the
+# rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+# sell copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+# IN THE SOFTWARE.
+"""
+Appengine Datastore backend.
+"""
+import ndb
+
+from flagon import errors
+from flagon.backends import Backend
+
+
+class FeatureToggle(ndb.Model):
+
+    enabled = ndb.BooleanProperty(indexed=False)
+    strategy_id = ndb.StringProperty('strategyId', indexed=False)
+    strategy_params_names = ndb.StringProperty('strategyParamsNames', indexed=False, repeated=True)
+    strategy_params_values = ndb.StringProperty('strategyParamsValues', indexed=False, repeated=True)
+
+
+class DatastoreBackend(Backend):
```

Comment:
nit: ``Datastore`` is a pretty generic term. I realize that is the name of the Google product, but it would be nice to make ``DatastoreBackend`` an alias to ``GoogleCloudDatastoreBakend`` or ``AppEngineDatastoreBackend``. That way we can slowly move it over to a more specific name without impacting your production instance(s). Example:

```python
class AppEngineDatastoreBackend(Backend):
    ...

#: Temporary alias.
DatastoreBackend = AppEngineDatastoreBackend
```